sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2560,6 +2560,33 @@ abstract class SQLQuerySuiteBase extends QueryTest with SQLTestUtils with TestHi
       }
     }
   }
+
+  test("SPARK-32818: make metastore convert session level configurable") {
+    withSQLConf(HiveUtils.CONVERT_METASTORE_ORC.key -> "true") {
+      withTable("t") {
+        sql("CREATE TABLE t (i INT) PARTITIONED BY (p STRING) STORED AS ORC")
+        sql("INSERT INTO t PARTITION(p='20200901') SELECT 1")
```

Comment:
I think we don't need the partition definition for this test and how about writing it like this;
```
sql("CREATE TABLE t (i INT) USING hive")
sql("SELECT * FROM t")
```